tools/python/descriptions/descriptions_downloader.py

```diff
@@ -138,7 +138,7 @@ def beautify_page(html, lang):
     for x in soup.find_all():
         if len(x.text.strip()) == 0:
             x.extract()
-    soup = remove_bad_sections(soup, lang)
+    # soup = remove_bad_sections(soup, lang)
```

Comment:
shall I remove all unused code or keep it commented in case we need it again?

Reply:
Зачем выпиливать все секции? Я вот читаю статьи, вроде реально всё интересно и нужно. Не лучше ли явно выпиливать только что-то нерелевантное? Какую проблему-то решаем?

Reply:
It leads to a significant maps size inflation (and its enabled for 5 languages only so far!), though added value for the most users is small.
I doubt many users will read **full** wikipedia articles in a map app, its not an offline wikipedia reader after all.

Many more users will read short descriptions of the POIs though (its kind of replacement of missing OSM descriptions) and given they take much less space the relative added value will be much higher.

Reply:
All articles I saw on the map were already stripped and contained only the summary or summary plus one or two other interesting and useful sections. That's why I'm asking, how much and what exactly can we save with this patch.

If you have some "bad" examples that are too large and can be easily stripped, because sections are "unnecessary", let's check them and maybe add these sections to exceptions.

Reply:
Check any capital or a significant sightseeing, e.g.
https://en.wikipedia.org/wiki/Eiffel_Tower
https://en.wikipedia.org/wiki/Tbilisi

Section names are non-standard, so it would be impossible to gain big savings by such filtering, e.g. I just checked some articles in a smaller russian city and found many sections of very little (or very specific) interest:
https://ru.wikipedia.org/wiki/Марийский_государственный_университет
https://ru.wikipedia.org/wiki/Йошкар-Олинская_ТЭЦ-2
https://ru.wikipedia.org/wiki/Йошкар-олинский_троллейбус

Reply:
Checked in the app. I see only a few relevant sections in all examples, except cities. We don't store wiki articles in the world map now, so it's impossible to check their sections.

Reply:
I dunno, what's useful in having sections with lists of universities' buildings and faculties or technical specs of the machinery used at electrical station...
I don't think its worth spending storage space and bandwidth for this stuff of special interest.

Reply:
I don't worry about MWM size, to be honest. Current wikipedia content in the app is good.